packages/pytest-simcore/src/pytest_simcore/simcore_services.py

```diff
@@ -34,26 +35,38 @@ def services_endpoint(
 
 @pytest.fixture(scope="function")
 async def simcore_services(
-    services_endpoint: Dict[str, URL], docker_stack: Dict
+    services_endpoint: Dict[str, URL], monkeypatch
 ) -> Dict[str, URL]:
     wait_tasks = [
-        wait_till_service_responsive(endpoint)
+        wait_till_service_responsive(
+            f"{endpoint}{SERVICE_HEALTHCHECK_ENTRYPOINT.get(service, '/v0/')}"
+        )
         for service, endpoint in services_endpoint.items()
     ]
     await asyncio.gather(*wait_tasks, return_exceptions=False)
+    for service, endpoint in services_endpoint.items():
+        monkeypatch.setenv(f"{service.upper().replace('-', '_')}_HOST", endpoint.host)
+        monkeypatch.setenv(
+            f"{service.upper().replace('-', '_')}_PORT", str(endpoint.port)
+        )
 
     yield
 
 
 # HELPERS --
-@tenacity.retry(**MinioRetryPolicyUponInitialization().kwargs)
+@tenacity.retry(
+    wait=tenacity.wait_fixed(5),

```

Comment:
Out of curiosity. Why did you drop all policies? you wanted different tuning of retry for tests?

Reply:
just to remove the depency to servicelib